src/redux/configureStore.js

```diff
@@ -0,0 +1,17 @@
+import { configureStore, combineReducers } from '@reduxjs/toolkit';
+import { applyMiddleware, compose } from 'redux';
+import thunk from 'redux-thunk';
+import rocketReducer from './rockets/rockets';
+import missionReducer from './missions/MissionSlice';
```

Comment:
- [ ] Please add redux-logger as one of your middlewares i.e `import logger from 'redux-logger';`  Install it with `npm i --save redux-logger`. It is a requirement for your project and it helps track errors from dispatched actions 👍 . Below is a screenshot showing the requirement mentioned in the documentation 
![image](https://user-images.githubusercontent.com/45460192/175381304-01fcc7d8-d5bc-4ab1-a52a-7007248580f0.png)